File: 2016/gara1/bicollatz/att/bicollatz.c
#include <stdio.h>
#include <assert.h>


int bicollatz(int A, int B) {
    // Mettete qui il codice della soluzione
    return 42;
}


int main() {
    FILE *fr, *fw;
    int A, B;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(2 == fscanf(fr, "%d %d", &A, &B));

    fprintf(fw, "%d\n", bicollatz(A, B));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed2/gara1/bicollatz/att/bicollatz.pas
function bicollatz(A, B: longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    bicollatz := 42;
end;


var
    A, B  : longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, A, B);

    writeln(fw, bicollatz(A, B));
    close(fr);
    close(fw);
end.
